test_common/messaging/cy.yml

```diff
@@ -364,74 +369,116 @@ cy:
       header: Deddfau sy’n berthnasol i’r gwasanaeth hwn
       content:
         - "Bydd y defnydd a wnewch o’r gwasanaeth hwn ac unrhyw anghydfod sy’n codi o’i ddefnyddio yn cael eu rheoli a’u dehongli yn unol â chyfreithiau Cymru a Lloegr, gan gynnwys, ond nid yn gyfyngedig i:"
-        - law_bullets:
-            - Deddf Camddefnyddio Cyfrifiaduron 1990
-            - Rheoliadau Cyffredinol Diogelu Data
-            - Deddf Galluedd Meddyliol 2005
+        - - Deddf Camddefnyddio Cyfrifiaduron 1990
+          - Rheoliadau Cyffredinol Diogelu Data
+          - Deddf Galluedd Meddyliol 2005
     how_to_use:
       header: Sut i ddefnyddio’r gwasanaeth hwn yn gyfrifol
       content:
         - Mae peryglon yn gysylltiedig â defnyddio cyfrifiadur sy'n cael ei rannu (e.e. mewn caffi rhyngrwyd), i ddefnyddio’r gwasanaeth hwn. Eich cyfrifoldeb chi yw bod yn ymwybodol o’r peryglon hyn ac osgoi defnyddio unrhyw gyfrifiadur lle mae posibilrwydd y gall eraill weld eich gwybodaeth bersonol. Chi sy’n gyfrifol os ydych chi’n dewis gadael cyfrifiadur heb ei ddiogelu tra byddwch yn defnyddio’r gwasanaeth hwn.
         - "Mae’n rhaid i chi gymryd gofal eich hunan i sicrhau nad yw’r ffordd yr ydych yn cael mynediad at y gwasanaeth hwn yn eich gadael yn agored i’r perygl o:"
-        - risk_bullets:
-            - feirysau
-            - cod cyfrifiadur maleisus
-            - neu ymyrraeth o fath arall a allai achosi difrod i’ch system gyfrifiadurol eich hunan
+        - - feirysau
+          - cod cyfrifiadur maleisus
+          - neu ymyrraeth o fath arall a allai achosi difrod i’ch system gyfrifiadurol eich hunan
         - Ni ddylech gamddefnyddio ein gwasanaeth drwy gyflwyno’n fwriadol firysau, cnafon (trojans), mwydod (worms), bomiau rhesymeg (logic bombs) neu unrhyw ddeunydd arall maleisus neu niweidiol i dechnoleg.
         - Ni ddylech geisio cael mynediad heb awdurdod at y gwasanaeth hwn, y system lle caiff ei storio nac unrhyw weinydd, cyfrifiadur neu gronfa ddata sy’n gysylltiedig ag o. Ni ddylech ymosod ar ein gwefan drwy ymosodiad gwrthod gwasanaeth neu ymosodiad gwrthod gwasanaeth a ddosbarthwyd.
     entering_sensitive_info:
       header: Nodi gwybodaeth bersonol sensitif
       content:
         - Mae’r gwasanaeth ar-lein hwn yn cynnwys nifer o feysydd testun rhydd lle fe’ch gwahoddir i nodi gwybodaeth bersonol sensitif.
         - "Os ydych yn rhoi gwybodaeth am eich:"
-        - information_bullets:
-            - credoau crefyddol
-            - cyfeiriadedd rhywiol neu’ch bywyd rhywiol
-            - tarddiad hiliol neu’ch tarddiad ethnig
-            - barnau gwleidyddol
-            - credoau crefyddol neu’ch credoau athronyddol
-            - aelodaeth undeb lafur
-            - data genetig neu ddata biometrig
-            - gwybodaeth am eich iechyd
+        - - credoau crefyddol
```

Comment:
in general - when testing content like this - you are proving that the words exist and not any form of structure - so the 'content' here could have just been an array of content which you loop through and check to see if it is present.

Reply:
i.e. you are not saying - 'Does this text exist and look exactly like this' - you are saying 'does this text exist' - and the only reason for doing that is to ensure it is available in the language you are testing with